Source/WebCore/dom/TemplateContentDocumentFragment.h

```diff
@@ -55,3 +55,8 @@ class TemplateContentDocumentFragment final : public DocumentFragment {
 };
 
 } // namespace WebCore
+
+SPECIALIZE_TYPE_TRAITS_BEGIN(WebCore::TemplateContentDocumentFragment)
+    static bool isType(const WebCore::Node& node) { return node.isDocumentFragment() && static_cast<const WebCore::DocumentFragment&>(node).isTemplateContent(); }
```

Comment:
Can this use `dynamicDowncast` and call the other `isType` overload instead of `static_cast` and repeating what it does?

Reply:
Sure, done.